utils/src/main/java/com/nedap/archie/util/KryoUtil.java

```diff
@@ -29,4 +38,9 @@ protected Kryo create () {
     public static Pool<Kryo> getPool() {
         return pool;
     }
+
+    @SuppressWarnings("rawtypes")
+    public static void addSerializer(Class clazz, Serializer serializer) {
+        serializers.put(clazz, serializer);
+    }
```

Comment:
```suggestion
    public static <T> void addSerializer(Class<T> clazz, Serializer<T> serializer) {
        serializers.put(clazz, serializer);
    }
```
This fixes the warning without suppressing it.